benchmark/process/bench-env.js

```diff
@@ -3,15 +3,55 @@
 const common = require('../common');
 
 const bench = common.createBenchmark(main, {
-  n: [1e5],
+  n: [1e6],
+  operation: ['get', 'set', 'enumerate', 'query', 'delete']
```

Comment:
I think this change means we need to add a line for `operation` in `test/benchmark/test-benchmark-process.js`.